### ISSUE TITLE
Install error

### ISSUE THREAD

**Issue by craigerskine** [stephanediondev/readerself#91]
````
Deprecated: Unparenthesized `a ? b : c ? d : e` is deprecated. Use either `(a ? b : c) ? d : e` or `a ? b : (c ? d : e)` in /.../system/helpers/url_helper.php on line 162
````

When I try to install for the first time I get this error. How do I fix this?

**Comment by stephanediondev** [comment]
Hi

This project is very old now and not updated.

The syntax is deprecated
https://www.php.net/manual/en/migration74.deprecated.php

Please have a look to a more recent version made with Symfony
https://github.com/stephanediondev/feed

**Comment by craigerskine** [comment]
Yeah, I tried installing feed but it did not work. I must be missing something.

I'm on a Mac and I tried using terminal and it looked like it finished, but there are no index.html/php files so I'm not sure what to upload.

I also tried on my Dreamhost server but the install failed saying I'm on PHP 7.3 but I'm not, all my sites are on PHP 7.4.

Super confusing.
